extra/utils.py

```diff
@@ -74,8 +74,9 @@ def find_class(self, module, name):
         elif name == "_rebuild_parameter":
           return HackParameter
       else:
+        if module.startswith('pytorch_lightning'): return Dummy
         try:
-          return pickle.Unpickler.find_class(self, module, name)
+          return super().find_class(self, module, name)
```

Comment:
`pytorch_lightning` does monkeypatching on Unpickler class, and it breaks `MyPickle`
https://github.com/Lightning-AI/lightning/blob/35d15621c7d007fa7418958b9eef120e14abc1a9/src/lightning/pytorch/_graveyard/legacy_import_unpickler.py#L46
```
Traceback (most recent call last):
  File "/home/kirill/3rdworkspace/tinygrad/extra/utils.py", line 92, in find_class
    cla = pickle.Unpickler.find_class(self, module, name)
  File "/home/kirill/miniconda3/envs/nnexp/lib/python3.10/site-packages/pytorch_lightning/_graveyard/legacy_import_unpickler.py", line 25, in find_class
    return super().find_class(new_module, name)
TypeError: super(type, obj): obj must be an instance or subtype of type
```